app/src/main/java/com/islaharper/dawnofandroid/presentation/components/LoadingEffect.kt

```diff
@@ -0,0 +1,127 @@
+package com.islaharper.dawnofandroid.presentation.components
+
+import android.content.res.Configuration
+import androidx.compose.animation.core.FastOutLinearInEasing
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Surface
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.ui.theme.DawnOfAndroidTheme
+import com.islaharper.dawnofandroid.ui.theme.PADDING_LARGE
+import com.islaharper.dawnofandroid.ui.theme.PADDING_MEDIUM
+import com.islaharper.dawnofandroid.ui.theme.PADDING_SMALL
+import com.islaharper.dawnofandroid.ui.theme.PADDING_XSMALL
+
+@Composable
+fun LoadingEffect(modifier: Modifier = Modifier) {
+    LazyColumn(
+        modifier = modifier.background(MaterialTheme.colorScheme.surface),
+        contentPadding = PaddingValues(all = PADDING_SMALL),
+        verticalArrangement = Arrangement.spacedBy(PADDING_MEDIUM),
+    ) {
+        items(count = 2) {
+            AnimatedLoadingItem()
+        }
+    }
+}
+
+@Composable
+fun AnimatedLoadingItem(modifier: Modifier = Modifier) {
+    val transition = rememberInfiniteTransition(label = "")
+    val alphaAnim by transition.animateFloat(
+        initialValue = 1f,
+        targetValue = 0f,
+        animationSpec = infiniteRepeatable(
+            animation = tween(
+                durationMillis = 650,
+                easing = FastOutLinearInEasing,
+            ),
+            repeatMode = RepeatMode.Reverse,
+        ),
+        label = stringResource(R.string.loading_animation_label),
```

Comment:
labels like this on the animation are developer-facing, I don't think you need to make a string resource for this (and incur the cost of translation surface area, separation etc.)